WorkloadManagementSystem/Client/SandboxStoreClient.py

```diff
@@ -266,14 +266,10 @@ def downloadSandboxForJob(self, jobId, sbType, destinationPath="", inMemory=Fals
     # If inMemory, ensure we return the newest sandbox only
     if inMemory:
       sbLocation = sbDict[sbType][-1]
-      result = self.downloadSandbox(sbLocation, destinationPath, inMemory, unpack)
-      return result
+      return self.downloadSandbox(sbLocation, destinationPath, inMemory, unpack)
 
     for sbLocation in sbDict[sbType]:
-      result = self.downloadSandbox(sbLocation, destinationPath, inMemory, unpack)
-      if not result['OK']:
-        return result
-    return S_OK()
+      return self.downloadSandbox(sbLocation, destinationPath, inMemory, unpack)
```

Comment:
This will end after the first entry in the list(?), and not keep looping over the rest of the locations.

Reply:
Thanks, I fixed.